src/AppStream.DurablePatterns/ActivityFunctions/ActivityFunction.cs

```diff
@@ -13,22 +13,21 @@ internal class ActivityFunction
         public const string FunctionName = "AppStream-DurablePatterns-ActivityFunction";
 
         private readonly IPatternActivityFactory _patternActivityFactory;
-        private readonly IStepConfigurationBag _stepConfigurationBag;
 
         public ActivityFunction(
-            IPatternActivityFactory patternActivityFactory,
-            IStepConfigurationBag stepConfigurationBag)
+            IPatternActivityFactory patternActivityFactory)
         {
             _patternActivityFactory = patternActivityFactory;
-            _stepConfigurationBag = stepConfigurationBag;
         }
 
         [FunctionName(FunctionName)]
-        public Task<ActivityFunctionResult> RunWorker(
-            [ActivityTrigger] IDurableActivityContext context)
+        public async Task<ActivityFunctionResult> RunWorker(
+            [ActivityTrigger] IDurableActivityContext context,
+            [DurableClient] IDurableEntityClient durableClient)
         {
             var functionInput = context.GetInput<ActivityFunctionInput>();
-            var stepConfiguration = _stepConfigurationBag.Get(functionInput.StepId);
+            var stepsResponse = await durableClient.ReadEntityStateAsync<StepsConfigEntity>(functionInput.StepsConfigEntityId);
```

Comment:
Maybe we could wrap it in something nicer? Like extension for DurableClient so that we could use:

```
durableClient.GetEntity<SetpsConfigEntity>(entityId)
```
?

Reply:
I don't get it, the only difference between:
```
durableClient.ReadEntityStateAsync<StepsConfigEntity>(entityId);
```
and
```
durableClient.GetEntity<SetpsConfigEntity>(entityId);
```
is the method name